src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
```

Comment:
Prefer using `const` over `let` for `result` as it does not need to be reassigned. Initialize it with the base cost calculation and then adjust the value as needed within the conditional statements.